site/en/docs/extensions/reference/commands/index.md

```diff
@@ -8,89 +8,159 @@ You must have a `"manifest_version"` of at least `2` to use this API.
+
+- Certain operating system and Chrome shortcuts (e.g. window management) always take priority over
+  Extension Command shortcuts and can not be overwritten.
+
+{% Aside 'note' %}
+
+Key combinations that involve `Ctrl+Alt` are not permitted in order to avoid conflicts with the
+`AltGr` key.
+
+{% endAside %}
+
+### Handling command events
 
 ```json
 {
   "name": "My extension",
   ...
   "commands": {
-    "toggle-feature-foo": {
+    "run-foo": {
       "suggested_key": {
         "default": "Ctrl+Shift+Y",
         "mac": "Command+Shift+Y"
       },
-      "description": "Toggle feature foo"
+      "description": "Run \"foo\" on the current page."
     },
-    "_execute_browser_action": {
+    "_execute_action": {
       "suggested_key": {
         "windows": "Ctrl+Shift+Y",
         "mac": "Command+Shift+Y",
         "chromeos": "Ctrl+Shift+U",
         "linux": "Ctrl+Shift+J"
       }
-    },
-    "_execute_page_action": {
-      "suggested_key": {
-        "default": "Ctrl+Shift+E",
-        "windows": "Alt+Shift+P",
-        "mac": "Alt+Shift+P"
-      }
     }
   },
   ...
 }
 ```
 
 In your background page, you can bind a handler to each of the commands defined in the manifest
-(except for '\_execute_browser_action' and '\_execute_page_action') via onCommand.addListener. For
-example:
+using `onCommand.addListener`. For example:
 
 ```js
-chrome.commands.onCommand.addListener(function(command) {
-  console.log('Command:', command);
+chrome.commands.onCommand.addListener((command) => {
+  console.log(`Command: ${command}`);
 });
 ```
 
-The '\_execute_browser_action' and '\_execute_page_action' commands are reserved for the action of
-opening your extension's popups. They won't normally generate events that you can handle. If you
-need to take action based on your popup opening, consider listening for an 'onDomReady' event inside
-your popup's code.
+### Action commands
+
+The `_execute_action` (Manifest V3), `_execute_browser_action` (Manifest V2), and
+`_execute_page_action` (Manifest V2) commands are reserved for the action of trigger your action,
+browser action, or page action respectively. These commands do not dispatch
+[command.onCommand][event-oncommand] events like standard commands.
+
+If you need to take action based on your popup opening, consider listening for a
+[DOMContentLoaded][html-dcl] event inside your popup's JavaScript.
 
 ## Scope
 
-By default, Commands are scoped to the Chrome browser, which means that while the browser does not
-have focus, the shortcut will be inactive. On desktop Chrome, Commands can instead have global
-scope, as of version 35, and will then also work while Chrome does \*not\* have focus. NOTE: The
-exception here is Chrome OS, where global commands are not allowed at the moment.
+By default, Commands are scoped to the Chrome browser. This means that when the browser does not
+have focus, Command shortcuts are inactive. Beginning in Chrome 35, extension developers can
```

Comment:
We do have other pages where we reference historical platform changes. I think we should leave the historical data as is for this PR and start a separate conversation about retention periods for this kind of information.

Is the highlighted text intended to focus on the casing used for "Command"? If so, I agree we should normalize this. At the moment it's inconsistent in this file. Unfortunately it's also inconsistent in the generated API docs, so we can't look to them for help.

Reply:
I wasn't commenting on the casing, which yes we should address later. I'm also fine with addressing the since Chrome nn later too. Go ahead and resolve this.